Small updates (various)

MeuhV1 opened the pull request:
Small pedia strategy text fixes
Skips Russian UP code if Russia isn't alive
Replaced some spaces with tabs

Commit:
Small pedia strategy text fixes
Skips Russian UP check if Russia isn't alive
Replaced some spaces with tabs

Commit:
More minor updates/fixes

MeuhV1 commented on Assets/Python/RFCUtils.py:
```diff
@@ -415,7 +415,7 @@ def flipUnitsInArea(self, tTopLeft, tBottomRight, iNewOwner, iOldOwner, bSkipPlo
+										if ((unit.getUnitType() > iHarappanCityBuilder)):
 											self.makeUnit(unit.getUnitType(), iNewOwner, [0, 67], 1)
 									else:
 										if ((unit.getUnitType() >= iSettler)): #skip animals
```
Shouldn't line 418 and 421 be swapped?

dguenms replied:
Yes. Can you also use unit.isFound() and unit.isAnimal() instead?

dguenms commented on Assets/Python/RFCUtils.py:
```diff
@@ -172,16 +172,16 @@ def isMortalUnit(self, unit):
 				if (gc.getGame().getSorenRandNum(100, 'random modifier') >= 50):
 					return False	      
 		iUnitType = unit.getUnitType()
-		if (iUnitType <= con.iKhmerBallistaElephant \
```
Good catch, but this method is a mess in general. I pushed a much cleaner implementation to develop, you might need to rebase though. Sorry about that.

dguenms commented on Assets/Python/RFCUtils.py:
```diff
@@ -700,7 +700,7 @@ def pushOutGarrisons(self, tCityPlot, iOldOwner):
 				if (unit.getDomainType() == 2): #land unit
 					unit.setXYOld(tDestination[0], tDestination[1])
 				else:
-					j = j + 1
```
Rhye and his C idioms :)

dguenms commented on Assets/Python/RiseAndFall.py:
```diff
@@ -2708,10 +2708,7 @@ def handleColonialConquest(self, iPlayer):
 					break
 
 		if tSeaPlot != -1:
-			if iPlayer == iNetherlands:
```
:+1:

dguenms commented on Assets/XML/Text/Units.xml:
```diff
@@ -989,7 +989,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_SCOUT_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Scouts[COLOR_REVERT] is able to enter [COLOR_HIGHLIGHT_TEXT]Jungle[COLOR_REVERT], [COLOR_HIGHLIGHT_TEXT]Rainforest[COLOR_REVERT] and [COLOR_HIGHLIGHT_TEXT]Marsh[COLOR_REVERT], gain only positive bonuses from Tribal villages, can only defend and have a combat bonus against [COLOR_UNIT_TEXT]Animal Units[COLOR_REVERT].</English>
```
Isn't the other verb wrong? Scouts is plural.

SgtBears commented on Assets/XML/Text/Units.xml:
```diff
@@ -844,7 +844,7 @@
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_OTTOMAN_JANISSARY_STRATEGY</Tag>
-		<English>The [COLOR_UNIT_TEXT]Janissary[COLOR_REVERT], the Ottoman replacement for the [COLOR_UNIT_TEXT]Musketman[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Archery Units[COLOR_REVERT], [COLOR_UNIT_TEXT]Mounted Units[COLOR_REVERT] and [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</English>
+		<English>The [COLOR_UNIT_TEXT]Janissary[COLOR_REVERT], the Turkish replacement for the [COLOR_UNIT_TEXT]Musketman[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Archery Units[COLOR_REVERT], [COLOR_UNIT_TEXT]Mounted Units[COLOR_REVERT] and [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</English>
```
What about the Ottoman Hammam?

SgtBears commented:
You changed the "ottoman" to "turkish" only for the janissary. You didn't change the Hammam. :)

MeuhV1 commented:
I didn't see scouts was plural. Will revert it. However, I just saw "Scouts is", which is wrong.
I'll add isFound and isAnimal.
Will also change Hammam to Turkish.

Commit:
Updates from comments

MeuhV1 commented:
Done.

Commit:
New art Floating Gardens
Highlighted "Scientist" in Arabian Alchemist UB (sync with other strategy texts)
Fixed text error in Victory.py (Harappan Granary -> Walls)
Brazilian UU ability also works on rainforest
Removed some tabs and spaces in Plague.py

MeuhV1 commented:
Update:

New art Floating Gardens
Highlighted "Scientist" in Arabian Alchemist UB (sync with other strategy texts)
Fixed text error in Victory.py (Harappan Granary -> Walls)
Brazilian UU ability also works on rainforest
Removed some tabs and spaces in Plague.py
